spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'takeaway'
+require 'menu'
+require 'kitchen'
+
+describe 'User Stories' do
+  let(:takeaway) { takeaway = TakeAway.new }
+  let(:kitchen) { kitchen = Kitchen.new }
+  let(:order) { double() }
+
+  context 'Menu class' do
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a list of dishes with prices
+    it 'receive menu' do
+      expect(takeaway.menu_list).not_to be nil
+    end
+
+    it 'should print list of dishes' do
+      expect { takeaway.menu_list }.to output.to_stdout
+    end
+  end
+
+  context 'Kitchen class' do
+    it 'should save order' do
+      kitchen.order "Tomato Salad"
+      expect(kitchen.order_cart).to eq kitchen.order_cart
```

Comment:
I think this will pass regardless of what you pass in - after the eq I would specify "Tomato Salad"